QEfficient/transformers/models/modeling_auto.py

```diff
@@ -35,10 +35,6 @@ class QEFFTransformersBase(QEFFBaseModel):
     _hf_auto_class: type
 
     def __init__(self, model: nn.Module) -> None:
-        model_class_name = model.__class__.__name__
-        if not (model_class_name.endswith("ForCausalLM") or model_class_name.endswith("LMHeadModel")):
```

Comment:
Yes we are still checking it, but now checking is happening in class`QEFFAutoModelForCausalLM`. Its good idea to check with the set of all the supported models but as we are not doing any check in the `QEffAutoModel`, it will be great if we make these changes with the new PR.